charts/azure-aks-aso/templates/credentials.yaml

```diff
@@ -20,7 +20,7 @@ stringData:
   {{- if .Values.clientSecret }}
   AZURE_CLIENT_SECRET: {{ .Values.clientSecret | quote }}
   {{- end }}
-  {{- if .Values.authMode }}
-  AUTH_MODE: {{ .Values.authMode | quote }}
+  {{- if .Values.useWorkloadIdentity }}
+  USE_WORKLOAD_IDENTITY_AUTH: {{ .Values.useWorkloadIdentity | quote }}
```

Comment:
This was correct before. `USE_WORKLOAD_IDENTITY_AUTH` is only valid in the global aso-controller-settings Secret where this is a per-resource Secret which has a slightly different format. The global aso-controller-settings secret defaults to managed identity unless `USE_WORKLOAD_IDENTITY_AUTH: true` is set and per-resource secrets like this default to workload identity unless `AUTHMODE: podidentity` is set.

https://azure.github.io/azure-service-operator/guide/authentication/credential-format/